src/pages/home/live-pricing/index.tsx

```diff
@@ -0,0 +1,158 @@
+import { handleRedirectToTradersHub } from 'components/layout/nav/util/nav-methods'
+
+const LivePricingSection = styled.section`
+    background-color: var(--color-white);
+    min-height: 780px;
+    display: flex;
+    flex-direction: column;
+    padding: 40px 0 80px;
+`
+
+const ContainerWrapper = styled(Flex)`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 36px;
+    margin: 0 auto;
+    padding: 2rem;
+    width: 60%;
+    @media ${device.tablet} {
+        width: 100vw;
+    }
+`
+
+const Markets = styled.div`
+    position: relative;
+    width: 100%;
+`
+
+const MarketsContainer = styled.div`
+    margin: 0 auto;
+    overflow-y: hidden;
+    overflow-x: auto;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    padding: 2rem;
+    scrollbar-width: none; /* Firefox */
+    ::-webkit-scrollbar {
+        display: none;
+    }
+    @media ${device.tablet} {
+        justify-content: flex-start;
+    }
+`
+const MarketButton = styled.button<{ selected: boolean }>`
+    margin: 0;
+    min-height: 48px;
+    position: relative;
+    border-bottom: 2px solid red;
+    background: none;
+    border: none;
+    padding: 16px 40px;
+    cursor: pointer;
+    transition: all 0.1s ease-in;
+    white-space: nowrap;
+
+    ${Header} {
+        font-weight: normal;
+        font-size: 2rem;
+        color: var(--color-grey-5);
+    }
+
+    ${({ selected }) =>
+        selected
+            ? css`
+                  border-bottom: 2px solid red;
+                  & ${Header} {
+                      color: var(--color-red);
+                  }
+              `
+            : css`
+                  border-bottom: 2px solid var(--color-grey-8);
+                  & ${Header} {
+                  }
+              `}
+`
+const DisclaimerText = styled(Header)`
+    color: var(--color-grey-5);
+    font-weight: normal;
+    text-align: center;
+    font-size: 1.6rem;
+`
+
+const LivePricing = () => {
+    const [selected_market, setSelectedMarket] = useState<TAvailableLiveMarkets>('forex')
+    const onMarketButtonClick = (selected) => {
+        setSelectedMarket(selected)
+    }
+    const [is_logged_in] = useAuthCheck()
+    const handleLogin = useHandleLogin()
+
+    return (
+        <LivePricingSection id="live-pricing">
+            <Markets>
+                <MarketsContainer>
+                    {market_buttons.map((marketItem) => (
+                        <>
+                            <MarketButton
+                                selected={marketItem.market_name === selected_market}
+                                key={marketItem.id}
+                                onClick={() => {
+                                    onMarketButtonClick(marketItem.market_name)
+                                }}
+                            >
+                                <Header type="paragraph-2">
```

Comment:
Don't forget to add `as="p"` property because it's h2 by default
And please take a look on each Header